enhancements/olm/simplify-apis.md

```diff
@@ -129,7 +129,7 @@ status:
   components:
    matchLabels:
       operators.coreos.com/operator:v2alpha1/plumbus: ""
```

Comment:
could you also correct this label convention? it's `operators.coreos.com/<operator-name>: ""` now. So in this case, we'd have `operators.coreos.com/plumbus`.